setup.py

```diff
@@ -4,6 +4,37 @@
 
 from setuptools import find_packages, setup
 
+# Fail with explicit error if trying to install Django 2 on Python 2, and
+# advise on how to avoid this.
+if sys.version_info < (3,):
+    sys.stderr.write(
+        """
+
+========
+WARNING!
+========
+
+You are running Django >= 2 on Python 2, but it requires Python 3.
+
+Unfortunately Django 2.0 and above are not compatible with Python 2,
+but you still ended up trying to install this version.  This should
+not have happened.  Make sure you have pip ≥ 9.0 to avoid this kind
+of issue, as well as setuptools ≥ 24.2:
```

Comment:
Does this character require a `coding:` comment on python 2?

Reply:
I don't actually know, but I'll switch to `>=` to be safe.

(yet another reason to be please that Python 2 is on the way out!)